content/account/dashboard/queues.textile

```diff
@@ -0,0 +1,35 @@
+---
+title: Queues
+meta_description: Manage and configure Ably queues, monitor realtime data, and optimize performance.”
+meta_keywords: “Ably Queues, realtime data, AMQP, STOMP, queue management, TTL, queue settings"
+---
+
+Ably queues provide a way to consume realtime data using the "AMQP":/general/queues#consume-amqp or "STOMP":/general/queues#consume-stomp protocols. Find out more about using "Ably queues":/general/queues#what.
+
+h2. Manage your Ably queues
+
+The  Ably queues tab enables you to:
+
+* Access a list of all your existing queues.
+* Monitor realtime data flow and queue performance.
+* Click on any queue to view and adjust its settings, such as TTL, maximum length, and region.
+
+h3. Provision a new queue
+
+When provisioning a new queue, you'll need to specify several things:
+
+|_. Field |_. Description |
+| *Name* | Choose a unique name for your queue. This will help you identify it within your dashboard and during application development. |
+| *Region* | Select the geographic region where the queue will be hosted. This is important for optimizing latency and ensuring data residency aligns with your application's requirements. |
+| *TTL (time to Live)* | Set the TTL, which determines how long messages remain in the queue before being automatically deleted if they are not consumed. The default account limit is 60 minutes. You can contact Ably support for assistance if you need a longer TTL. |
+| *Max length* | Define the maximum number of messages the queue can hold at any given time. The default limit is 10,000 messages, but you can request an increase if your application requires more capacity. |
+
+<aside data-type='note'>
+<p>If your application requires more than the default limits (e.g. longer TTLs or more messages), contact "Ably support":https://ably.com/support to discuss custom configurations. Find out more about "provisioning a new queue":/general/queues#provisioning.</p>
+</aside>
```

Comment:
I'd drop this (it needs to be removed from the Queues page too and consolidated under limits.

Reply:
I've removed